### ISSUE TITLE
Bumping ABI version to 1?

### ISSUE THREAD

**Issue by matze** [ufo-kit/ufo-core#164]
Just a general question also towards @tfarago what is necessary to bump the ABI version and release a version 0.17. From my point of view we can just do it and then let Debian handle packaging of the new version.

**Comment by tfarago** [comment]
I don't see any issues with that. What does it actually contain for us?

**Comment by matze** [comment]
We broke the ABI by removing symbols (remember?) and it's Debian policy and good manner to increase the ABI version to avoid confusion and anger.

**Comment by tfarago** [comment]
Nope... What symbols? But if that's the case then sure, let's bump it.

**Comment by matze** [comment]
[Those](https://github.com/ufo-kit/ufo-core/commit/aedece915a5338865ee0f650c6f4cace5f0bb2e8) for example ;-)

**Comment by tfarago** [comment]
Ahaaaaa

**Comment by stuart-cls** [comment]
Now that the Debian packages are being updated (https://tracker.debian.org/pkg/ufo-core) it would be nice to have this so that `tofu` can be used against locally built or packaged versions of ufo without having to mangle the `gi.require_version('Ufo', '1.0')` lines.

**Comment by tfarago** [comment]
Actually we did recently what @matze proposed 7 years (!) ago... The package version is 0.17 and ABI has been 1.0 for a while. @stuart-cls can you please elaborate what the problem is? I am sensing something about warnings about the version when you do not do `gi.require_version`?

**Comment by stuart-cls** [comment]
If you install ufo-core from source, you must use (in Python) `gi.require_version('Ufo', '0.0')` but if you install it from the Debian packages it is spelled `gi.require_version('Ufo', '1.0')`. So I think the source version did not actually bump the gobject namespace versioning.
